inject-generator/src/test/java/io/avaje/inject/generator/InjectProcessorTest.java

```diff
@@ -67,7 +68,7 @@ void testGeneration() throws Exception {
             List.of("--release=" + Integer.getInteger("java.specification.version")),
             null,
             files);
-    task.setProcessors(Arrays.asList(new InjectProcessor()));
+    task.setProcessors(List.of(new InjectProcessor(), new ServiceProcessor()));
```

Comment:
@rbygrave you can probably use your IDE here to debug and understand what's going on if it's still not making intuitive sense.